### PR TITLE
CRAN release 0.4.0

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -25,7 +25,7 @@ BugReports: https://github.com/easystats/datawizard/issues
 Depends:
     R (>= 3.4)
 Imports:
-    insight (>= 0.16.0.5),
+    insight (>= 0.16.9.0),
```
Comment (strengejacke) [inline DESCRIPTION:28]: 0.17.0 is now on CRAN

**Comment by strengejacke** [comment]
_insight_ is on CRAN now, I think _datawizard_ should be next, followed by _modelbased_? @DominiqueMakowski

File: DESCRIPTION (hunk 1 above)
Comment (IndrajeetPatil) [reply DESCRIPTION:28]: I can't work on this during office hours. 

Can you please make this change yourself and let me know? I can then submit it to CRAN.

**Comment by strengejacke** [comment]
Checks may fail, because version 0.17.0 is not yet published on mirrors. This means that win-builder will probably not yet work, if insight 0.17.0 is required and no win-binaries built yet.

File: DESCRIPTION [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: datawizard
 Title: Easy Data Wrangling
-Version: 0.3.1.9010
+Version: 0.4.0
```
Comment (IndrajeetPatil) [inline DESCRIPTION:4]: This also needs to be changed in `NEWS.md`.

**Comment by IndrajeetPatil** [comment]
Feel free to merge this.

**Comment by IndrajeetPatil** [comment]
Sounds good!

**Comment by IndrajeetPatil** [comment]
```r
* using log directory 'd:/RCompile/CRANguest/R-devel/datawizard.Rcheck'
* using R version 4.2.0 alpha (2022-03-28 r82012 ucrt)
* using platform: x86_64-w64-mingw32 (64-bit)
* using session charset: UTF-8
* checking for file 'datawizard/DESCRIPTION' ... OK
* checking extension type ... Package
* this is package 'datawizard' version '0.4.0'
* package encoding: UTF-8
* checking CRAN incoming feasibility ... NOTE
Maintainer: 'Indrajeet Patil <patilindrajeet.science@gmail.com>'

Found the following (possibly) invalid URLs:
  URL: https://easystats.github.io/datawizard/articles/demean.html
    From: man/demean.Rd
    Status: 404
    Message: Not Found
* checking package namespace information ... OK
* checking package dependencies ... ERROR
Package required and available but unsuitable version: 'insight'

See section 'The DESCRIPTION file' in the 'Writing R Extensions'
manual.
* DONE
Status: 1 ERROR, 1 NOTE
```

**Comment by strengejacke** [comment]
Do we need the `card.png`? Is it used somewhere? I think it takes roughly 75% of the package size.
https://github.com/easystats/datawizard/tree/master/man/figures

**Comment by IndrajeetPatil** [comment]
We use it for website:
https://github.com/easystats/datawizard/blob/c97eed4196703f6d42f6f97b4a5261cc278b6488/_pkgdown.yml#L28-L30

@DominiqueMakowski Is there a way to reduce size of these cards?

**Comment by strengejacke** [comment]
> We use it for website:

But where is it shown?

**Comment by IndrajeetPatil** [comment]
You will see it if you share the website URL on a social media platform like Twitter.

You can see for yourself using [Twitter Card Validator](https://cards-dev.twitter.com/validator).

**Comment by IndrajeetPatil** [comment]
You can remove it for the submission. We can figure out what to do with it later.

**Comment by strengejacke** [comment]
ok, but why don't we use the existing datawizard logo? And it doesn't seem to work, anyway, so away with it! ;-)
image

**Comment by strengejacke** [comment]
submitted. win-builder will still fail due to older binaries for insight, so we hope that everything is ok on that side :-)

**Pull request merged**